operation_functions.py

```diff
@@ -0,0 +1,26 @@
+MAJOR_COLORS = ['White', 'Red', 'Black', 'Yellow', 'Violet']
+MINOR_COLORS = ["Blue", "Orange", "Green", "Brown", "Slate"]
+
```

Comment:
Think of a more specific filename than `operation_functions` - like you've done for `ReferenceManual`